python/lsst/consdb/transformed_efd/config_model.py

```diff
@@ -0,0 +1,99 @@
+"""Defines the configuration models used in the application."""
```

Comment:
Please include the standard headers: https://developer.lsst.io/stack/license-and-copyright.html

Reply:
Acknowledged, I'll implement that.